Index: psychopy/tools/gltools.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- psychopy/tools/gltools.py	(date 1570200959000)
+++ psychopy/tools/gltools.py	(date 1575032192000)
@@ -89,8 +89,7 @@
 import warnings

 # create a query counter to get absolute GPU time
-QUERY_COUNTER = GL.GLuint()
-GL.glGenQueries(1, ctypes.byref(QUERY_COUNTER))
+QUERY_COUNTER = None


 # compatible Numpy and OpenGL types for common GL type enums
@@ -1086,6 +1085,11 @@
         timeElapsed = (t1 - t0) * 1e-9  # take difference, convert to seconds

     """
+    global QUERY_COUNTER
+    if QUERY_COUNTER is None:
+        QUERY_COUNTER = GL.GLuint()
+        GL.glGenQueries(1, ctypes.byref(QUERY_COUNTER))
+
     GL.glQueryCounter(QUERY_COUNTER, GL.GL_TIMESTAMP)

     params = GL.GLuint64(0)
